tests/xtimer_drift/main.c

```diff
@@ -89,6 +89,7 @@ void *worker_thread(void *arg)
     (void) arg;
     unsigned int loop_counter = 0;
     uint32_t start = 0;
+    uint32_t last = 0;
```

Comment:
Yes please. Prevents others from taking time to make their mind about the distinction. :-)